packages/pyinstaller/bitmessagemain.spec

```diff
@@ -72,6 +72,13 @@ a.datas += [
     for file_ in os.listdir(dir_append) if file_.endswith('.ui')
 ]
 
+sql_dir_append = os.path.join(srcPath, 'sql')
```

Comment:
I think you can use  `glob`, that simplifies it.

Reply:
Is it like globe('sql') ?

Reply:
```sql_dir_append = glob(os.path.join(srcPath, 'sql/*.sql'))```

Reply:
sql_dir_append = glob(os.path.join(srcPath, 'sql/*.sql'))